src/groupeddataframe/groupeddataframe.jl

```diff
@@ -323,6 +323,20 @@ function Base.to_index(gd::GroupedDataFrame, key::NamedTuple{N}) where {N}
     return Base.to_index(gd, Tuple(key))
 end
 
+function _dict_to_tuple(key::Dict{T, S}, gd) where {T, S}
```

Comment:
As below I think it is better to be stricter just to be sure all works OK.
```suggestion
function _dict_to_tuple(key::Union{Dict{Symbol},Dict{<:AbstractString}} , gd)
    T = keytype(d)
```